libtenzir/src/tql2/exec.cpp

```diff
@@ -411,11 +411,19 @@ auto exec2(std::string_view source, diagnostic_handler& dh,
       // Do not proceed to execution if there has been an error.
       return false;
     }
-    auto result = exec_pipeline(std::move(pipe), ctx, cfg, sys);
-    if (not result) {
-      diagnostic::error(result.error()).emit(ctx);
+    for (auto pipe : std::move(pipe).split_at_void()) {
```

Comment:
We discussed that we might want to make this opt-in, e.g., via `tenzir --multi '<pipelines>'`.